metricbeat/module/docker/container/data.go

```diff
@@ -31,7 +32,26 @@ func eventMapping(cont *dc.APIContainers) common.MapStr {
 		"status": cont.Status,
 	}
 
+// Check id container have health metrics configured
+if strings.Contains(cont.Status, "(") && strings.Contains(cont.Status, ")") {
+	container, _ := m.dockerClient.InspectContainer(cont.ID)
```

Comment:
Indeed if we have 100 containers with healthcheck enabled it could take some more time.
In order to get these Container Health information precisely, we have to inspect the container, it couldn't be fetch easily so the InspectContainer function. How clould we make container.health option ? as metricbeat docker module ? (like container, disk, cpu ...)